install_config/install/advanced_install.adoc

```diff
@@ -2215,57 +2205,72 @@ ifdef::openshift-origin[]
 
 . Use the `systemctl` command to start the installer service as you would any
-other systemd service. This command initiates the cluster installation:
+other systemd service. This command initiates the cluster installation using
+the inventory file specified above and the root user's SSH configuration:
++
+----
+# systemctl start openshift-installer
+----
++
+There is no output from this command while the installer service is
+running. Instead, output from the installer is logged to the system
+journal. You can follow the log using *journalctl*:
 +
 ----
-$ systemctl start openshift-installer
+# journalctl -f -o cat -u openshift-installer
 ----
 +
 If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific instructions
 or workarounds.
 
-. After the installation completes, you can uninstall the system container if you want. However, if you need to run the installer again to run any other playbooks later, you would have to follow this procedure again.
+. After the installation completes, you can uninstall the service if
+you want. To run the installer with different parameters (for instance
+to run any other playbooks), you will need to follow this procedure
+again. You can either uninstall the service and re-install it or just
+install a service using a different name.
 +
-To uninstall the system container:
+To uninstall the installer service:
 +
 ----
-$ atomic uninstall openshift-installer
+# atomic uninstall openshift-installer
 ----
 
 [[running-the-advanced-installation-system-container-other-playbooks]]
 ==== Running Other Playbooks
 
-After you have completed the cluster installation, if you want to later run any
-other playbooks using the containerized installer (for example, cluster upgrade
-playbooks), you can use the `PLAYBOOK_FILE` environment variable. The default
-value is `playbooks/byo/config.yml`, which is the main cluster installation
-playbook, but you can set it to the path of another playbook inside the
-container.
+To run any other playbooks (for example, to run the
+xref:configuring-cluster-pre-install-checks[pre-install checks]
+before proceeding to install) using the containerized installer, you can
+specify a playbook with the `PLAYBOOK_FILE` environment variable. The
+default value is `/usr/share/ansible/openshift-ansible/playbooks/byo/config.yml`, which is the main cluster
+installation playbook, but you can set it to the path of another playbook
+inside the container.
 
 For example:
 
 ----
-$ atomic install --system \
+# atomic install --system \
     --storage=ostree \
     --name=openshift-installer \
-    --set INVENTORY_FILE=/etc/ansible/hosts \
-    --set PLAYBOOK_FILE=playbooks/byo/openshift-cluster/upgrades/v3_6/upgrade.yml \//<1>
+    --set INVENTORY_FILE=/path/to/inventory \
+    --set PLAYBOOK_FILE=/usr/share/ansible/openshift-ansible/playbooks/byo/openshift-checks/pre-install.yml \//<1>
```

Comment:
As currently implemented, you actually need the full path for the playbook. This will change "soon" with updates to the image and we'll be able to go back to relative paths (the full path should always work though).